spec/support/step_nav_steps.rb

```diff
@@ -106,6 +106,12 @@ def given_there_is_a_step_by_step_page_with_broken_links_and_unpublished_changes
     @step_by_step_page = create(:step_by_step_page, steps: [step], draft_updated_at: Time.zone.now, slug: "step-by-step-with-broken-links-and-unpublished-changes")
   end
 
+  def given_there_is_a_step_by_step_page_submitted_for_2i
+    step = create(:step)
+    stub_link_checker_report_success(step)
+    @step_by_step_page = create(:step_by_step_page_submitted_for_2i, steps: [step], slug: "step-by-step-with-link-report")
+  end
```

Comment:
This doesn't appear to be being used anywhere (and also doesn't appear to create a step by step page with `submitted_for_2i` status). Can it be removed?

Reply:
Yes, this is not used anymore as we can now create a `:step_by_step_page_submitted_for_2i`. Removed.